src/App.js

```diff
@@ -0,0 +1,56 @@
+import './App.css';
+import React, { Component } from 'react';
+
+import Solution from './components/Solution'
+import Letters from './components/Letters'
+import Score from './components/Score'
+import EndGame from './components/EndGame';
+
+export class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      letterStatus: this.generateLetterStatuses(),
+      solution: { word: 'CALM', hint: 'Your ideal mood when coding.' },
+      score: 100,
+      gameEnded: false,
+      won: false
+    }
+  }
+
+  generateLetterStatuses() {
+    let letterStatus = {}
+    for (let i = 65; i < 91; i++) {
+      letterStatus[String.fromCharCode(i)] = false
+    }
+    return letterStatus
+  }
+
+  selectLetter = (letter) => {
+    let score, gameEnded, won
```

Comment:
maybe declare them at the top so there is not so much `this`

`const {solution, letterStatus, score} = this.state`
then you will not have to keep repeating `this.state`